scroll-animations-1/Overview.bs

```diff
@@ -853,17 +884,44 @@ instead of document's scrolling Element
 <pre class='descdef'>
   Name: start
   For: @scroll-timeline
-  Value: auto | <<length-percentage>>
+  Value: <<scroll-timeline-offset>>
   Initial: auto
 </pre>
 
 'start' descriptor determines the scroll timeline's {{start}}.
 
+[=Scroll timeline offsets=] in CSS are represented by the
+<<scroll-timeline-offset>> type:
+
+<pre>
+<dfn>&lt;scroll-timeline-offset></dfn> = auto | <<length-percentage>> | <<element-offset>>
+<dfn>&lt;element-offset></dfn> = selector(<<id-selector>>) [<<element-offset-edge>> || <<number>>]?
```

Comment:
This is totally my bad, since my suggestion had the same mistake, but we normally want `function( <foo> )` rather than `function(<foo>)`. See the [math function syntax](https://drafts.csswg.org/css-values-4/#calc-syntax) for reference.

The 'source' descriptor has the same issue, by the way.

Reply:
Good catch. I fixed the syntax for selector.